tests/initialize.js

```diff
@@ -106,6 +108,8 @@ const initialize = async (port, settingsDataset) => {
 
   await broker.createService(AnnouncerService);
 
+  await broker.createService(ProfilesApp);
```

Comment:
I think this service is manually loaded in the contacts.test.js file, be sure to remove it there.